src/containers/Header/NetworkSelector/NetworkSelector.scss

```diff
@@ -0,0 +1,146 @@
+@import '../../shared/css/variables';
+
+@mixin dropdown-network-item($background, $border: $black, $color: $white) {
+  background: $background;
+
+  .dropdown {
+    border-color: $border;
+    background: $background;
+    color: $color;
+
+    .item {
+      background: $background;
+    }
+  }
+}
+
+.network {
+  display: flex;
+  height: 23px;
+  padding: 0 16px;
+  margin: 0 -16px;
+
+  .arrow-container {
+    display: flex;
+    width: 100%;
+    align-items: center;
+
+    .arrow {
+      width: 10px;
+      color: $white;
+
+      &.open {
```

Comment:
Why does this include the & when other lines don't? (Does this mean that it needs to have both `arrow` and `open` in the same element instead of parent / child relationship?

Reply:
Yes it means only apply the styling when they are on the same class level